scripts/generate-docs-post-purchase.ts

```diff
@@ -12,7 +12,8 @@ const postPurchase = {
   },
   outputRoot: '../shopify-dev/content/api/checkout-extensions',
   shopifyDevUrl: '/api/checkout-extensions',
-  shopifyDevAssets: '../shopify-dev/app/assets/images/api/checkout-extensions',
+  shopifyDevAssets: '../shopify-dev/app/assets/images/api/post-purchase',
+  shopifyDevAssetsUrl: '/assets/api/post-purchase',
```

Comment:
I like this flatter structure, but I think only including `checkout-ui` assets under `checkout-extensions` could be misleading. What are your thoughts on dropping pp assets at `/assets/api/checkout-extensions/post-purchase`? I could look at updating the script so that .md files are created at `/content/api/checkout-extensions/post-purchase` (not for Editions, but afterwards)

Reply:
That works for me.

```
assets
|-- api
     |-- checkout-extensions
         |-- checkout
              |-- components
         |-- post-purchase
              |-- components
```